README.md

```diff
@@ -124,11 +124,13 @@ binci -e "/bin/sh"
 
 The above would start the container using the configuration, call the `before` task, then start the `sh` shell. The container will then remain in the shell until an `exit` command is sent by the user.
 
-## Container Image (`from <string>`)
+## Container Image (`dockerfile <string>` or `from <string>`)

```

Comment:
This is going to seem so nit-picky it's gross, but can you reverse this so `from` is the first topic, `dockerfile` the second. My reasoning is; 1. `binci init` creates with `from`, and 2. there hasn't been a voiced need for this until now (at least not one acted upon) so I think the majority use-case is `from`.

Reply:
Hahaha, flipped in latest push. Though, to counterpoint (because I have to :D)

1. init also writes `dockerfile` now in addition to `from`.
2. I think the only reason the majority use case is `from` is because that's the habit we instilled. Testing your own Dockerfile/running tasks against a prod-like build is a generally Good Thing (how many times did we forget to update something in our Dockerfile in the past couple years and crash staging), and needing an extra step of maintaining an external base image for a project that requires anything custom at all is a step we got used to/comfortable with, but it's additional work that's now solved by just putting that Dockerfile in your project repo and not using `from` anymore. This repo right here is a great example. The repos at our former employer could have made sense under this model too, considering some of them had to run `apk add` before most tasks and others just included their deps in the image we maintained globally. I think it's friggin phenomenal that Binci doesn't force you to build if you don't want to, but I think building by default is one of the things Compose got right. Handling how and when that happens, not so much... but the build is good.